analysis/aws_rules_cis/aws_console_login_without_mfa.yml

```diff
@@ -1,6 +1,6 @@
 AnalysisType: rule
 Filename: aws_console_login_without_mfa.py
-PolicyID: AWS.CloudTrail.ConsoleLoginWithoutMFA
+PolicyID: AWS.Console.LoginWithoutMFA
```

Comment:
I think the pattern we're following is that the prefix is the type of resource which is typically evaluated? We don't have an `AWS.Console` resource. Deferring to @nhakmiller on the matter

Reply:
These are rules, so resource doesn't apply.

Reply:
Ah yeah 👍

Reply:
Initially I was going to agree with Austin, for policies we follow the pattern of `AWS.Service.ResourceType.Description`, occasionally with `AWS.ResourceType.Description` if the service is easily implied, and with Logs we follow the pattern of `AWS.Logtype.Description`. But looking back it appears we diverge from this pattern in a lot of places actually, generally to prefer a more readable description. I'm ok with these changes, but we should be aware in the future that changing policy ID's makes it difficult to do updates because the old policies will stick around. Perhaps after panther goes v1 we should refrain from many policy ID changes.